src/documents/APIChangesFrom1_5To2_0.md

```diff
@@ -48,14 +47,15 @@ fluid.default("fluid.parent", {
 
 ##### In 2.0 #####
 
-In Infusion 2.0 where relay components are introduced, sharing models no longer requires the change applier to be shared:
+In Infusion 2.0 where relay components are introduced, the change applier must not be configured separately - model sharing
```

Comment:
It would be good if "change applier" here was a link to the ChangeApplier documentation. Also, perhaps instead of "must not" maybe "doesn't need to be"?